common4j/src/main/com/microsoft/identity/common/java/commands/parameters/nativeauth/SignInStartUsingPasswordCommandParameters.java

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+package com.microsoft.identity.common.java.commands.parameters.nativeauth;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.experimental.SuperBuilder;
+
+/**
+ * a set of Sign In Start command parameters for sending the start request to trigger the sign in flow but using email password.4
+ * extends from {@link SignInStartCommandParameters}
+ */
+@Getter
+@EqualsAndHashCode(callSuper = true)
+@SuperBuilder(toBuilder = true)
+public class SignInStartUsingPasswordCommandParameters extends SignInStartCommandParameters {
+    private static final String TAG = SignInStartUsingPasswordCommandParameters.class.getSimpleName();
+
+    /**
+     * The password of the user.
+     */
+    @NonNull
+    public final String password;
```

Comment:
Discussed offline. This is a security bug. Passwords should not be kept in Strings, instead a `char` array should be used. This needs to be fixed ASAP.

Reply:
I have used char array instead of string for password. Wherever the char array is created, it is zero'd out after usage.